valor.css

```diff
@@ -129,3 +136,14 @@ namaka-archive-comment::part(bold),
 namaka-ban::part(bold) {
 	color: #000;
 }
+
+namaka-comments {
+	/* border: none; */
+	padding: 0;
+}
+
+namaka-comment::part(btn) {
```

Comment:
prefer to use full words like `button` instead of acronyms or shorthands like `btn`

Reply:
Fixed